bentoml/_internal/service/service.py

```diff
@@ -233,19 +233,19 @@ def add_asgi_middleware(
     def get_grpc_servicer(self) -> service_pb2_grpc.BentoServiceServicer:
```

Comment:
The implementation needs to change based on the changes in `.proto` files. I suggest something like this:
```  
   def get_grpc_servicer(self) -> service_pb2_grpc.BentoServiceServicer:

        from bentoml.io import Text
        from bentoml.io import NumpyNdarray
        from bentoml.grpc import payload_pb2
        from bentoml.grpc import service_pb2
        from bentoml.grpc import service_pb2_grpc

        apis = self.apis

        class BentoServiceServicer(service_pb2_grpc.BentoServiceServicer):
            async def Call(
                self,
                request: service_pb2.CallRequest,
                context: grpc.ServicerContext,
            ) -> service_pb2.CallResponse:
                try:
                    api = apis[request.api_name]
                except BentoMLException:
                    raise BentoMLException(
                        f'Provided api_name "{request.api_name}" is not defined in service."'
                    )
                input = request.input

                io_type = input.WhichOneof("kind")

                if io_type == "string_value":
                    input = input.string_value
                elif io_type == "array_value":
                    input = input.array_value
                    input = await api.input.from_grpc_request(input, context)
                else:
                    raise ValueError("Invalid input type.")

                output = api.func(input)

                if isinstance(api.output, Text):
                    output = payload_pb2.Value(string_value=output)
                elif isinstance(api.output, NumpyNdarray):
                    out: payload_pb2.Array = await api.output.to_grpc_response(output)
                    output = payload_pb2.Value(array_value=out)
                return service_pb2.CallResponse(output=output)

        return BentoServiceServicer```